docker/docker-compose-valgrind.yml

```diff
@@ -1,34 +1,36 @@
-version: "3.7"
+versioe: "3.8"
```

Comment:
@little-dude Do we still want to keep the `valgrind` stuff?

Reply:
Probably not. But it doesn't hurt so I didn't comment about it.

Reply:
If not, please do let me know, otherwise I just get rid of it.

Reply:
Let's get rid of it then.

Reply:
Ok, will get rid of it, then!